### PR TITLE
Fix cursors leak with APM

### PR DESCRIPTION
**Pull request by martinkuba**
Please see this issue in JIRA for details and a repro
https://jira.mongodb.org/browse/NODE-889

I am not sure if this is the right fix, but wanted to at least get feedback / start discussion.

### CODE REVIEW

File: lib/apm.js [hunk 1]
```diff
@@ -342,7 +342,7 @@ var Instrumentation = function(core, options, callback) {
           }
 
           if(cmd.maxTimeMS) command.maxTimeMS = cmd.maxTimeMS;
-        } else if(x == '_killcursors') {
+        } else if(x == '_killcursor') {
```
Comment (martinkuba) [inline lib/apm.js]: This does not make a difference for fixing this particular issue, but the methods array above has '_killcursor' (not plural).

File: lib/apm.js [hunk 2]
```diff
@@ -461,6 +461,8 @@ var Instrumentation = function(core, options, callback) {
               reply: [{ok:1}]
             };
 
+            func.apply(this, arguments);
```
Comment (martinkuba) [inline lib/apm.js]: This seems to be the issue - the command object is created, but the original function is not invoked.

**Comment by christkv** [comment]
Fixed in b034c8399148a28d3d51f795c5157c835d077ee4

**Comment by christkv** [comment]
merged yours and added test

**Pull request closed without merging**